### ISSUE TITLE
Importing withResponsiveness() helper

### ISSUE THREAD

**Issue by dupski** [eventbrite/britecharts-react#144]
Hi Guys. Could you advise on the intended way to import the `withResponsiveness` helper?

## Current Behaviour

In the docs it uses a relative import, which I assume is because you compile your docs examples (nice)

<img width="715" alt="screen shot 2018-12-05 at 7 37 41 am" src="https://user-images.githubusercontent.com/8173903/49471327-b1e62900-f860-11e8-99a4-4ce90480110a.png">

I've found the compiled version in `britecharts-react/lib/esm/helpers` and the below code works fine, but is quite long-winded:

```ts
import withResponsiveness from 'britecharts-react/lib/esm/helpers/withResponsiveness';
```

## Expected Behaviour

I would like to import `withResponsiveness()` from the root, so:

```ts
import { Bar, withResponsiveness } from 'britecharts-react';
```

## Possible Solution

Happy to submit a PR to fix this, just let me know :)

## Your Environment

* Version used: `0.4.1` with TypeScript 3.1 and React 16.6

**Comment by Golodhros** [comment]
Hi @dupski, thanks for opening this issue!

Well, we weren't planning to include that with the library, that's why is so hard to access. That said, this is not the first time that somebody asks for it.

Should we include it in our dist folder then?

**Comment by davegomez** [comment]
Since is an option we have for the charts and is a feature listed in the docs I think it should be included or at least well referenced in the docs, but first I think we should fix the issue #105 that is still happening in `britecharts-react` v0.5.0 as you can confirm in this [test project](https://codesandbox.io/s/43101lzyw9).

**Comment by miglesiasEB** [comment]
All ready and working properly!

**Comment by juanpaucar** [comment]
@miglesiasEB Hi guys, sorry to bother you. I was wondering if you have a date to upload this to npm. I'm having some issue with this on a typescript project and jest.... and I ran out of ideas to make it work for the tests

**Comment by Golodhros** [comment]
Hi @juanpaucar,

I just released 0.5.1 with this fix, it should be ready in NPM soon!

**Comment by juanpaucar** [comment]
Thank you very much :D
